src/pipecat/processors/aggregators/llm_response.py

```diff
@@ -36,6 +37,51 @@
 from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
 
 
+class LLMFullResponseAggregator(FrameProcessor):
+    """This is an LLM aggregator that aggregates a full LLM completion. It
+    aggregates LLM text frames (tokens) received between
+    `LLMFullResponseStartFrame` and `LLMFullResponseEndFrame`. Every full
+    completion is returned via the "on_completion" event handler.
+
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        self._aggregation = ""
+        self._started = False
+
+        self._register_event_handler("on_completion")
+
+    async def process_frame(self, frame: Frame, direction: FrameDirection):
+        await super().process_frame(frame, direction)
+
+        if isinstance(frame, StartInterruptionFrame):
```

Comment:
Could it be worth sending what we have aggregated so far in this case, before we clean it? Maybe, along with the aggregation, we could also include information on whether it was completed or interrupted.

I have no strong feelings about this, just wondering if you see any use case where this could be useful.

Reply:
Good idea! I can add an argument that is `complete: bool` or `final: bool`.

Reply:
@filipi87 Please, take a look again.

Reply:
The only thing I was thinking about is the variable name. I'm wondering if it's clear enough or if it could cause some confusion. 

Maybe it would help to improve the documentation to specify when `final` will be `true` or `false`.

Or even be more verbose, like: `aggregation_status`: `completed` or `interrupted`.

What do you think ?

Reply:
OK, as discussed. Changed from `final` to `completed`. Thank you!